### PR TITLE
Story/io 227/project hashtags model endpoint

### PR DESCRIPTION
**Pull request by Git-Uzair**
Created  ProjectHashTag model, endpoint.
changed hashTags field on the Project model from jsonFIeld to ManyToManyField
Modified Project Tests to accomodate testing this new field.
Migration for new model created including data population for predefined hashtags

### CODE REVIEW

File: infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.1.3 on 2023-01-18 10:59
+
+from django.db import migrations, models
+import uuid
+
+
+def populate_ProjectHashTags_data(apps, schema_editor):
```
Comment (kgunga) [inline infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py:7]: Rename this to populateProjectHashTags

File: infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    ProjectHashTag = apps.get_model("infraohjelmointi_api", "ProjectHashTag")
+    projectHashtags = [
```
Comment (kgunga) [inline infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py:9]: Add here some comments about these hashtags. What are these ones? Most popular?

File: infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.1.3 on 2023-01-18 10:59
+
```
Comment (kgunga) [inline infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py:2]: Rename migration file to more describable name. 0024_redefine_project_hashtags_and_create_hashtags_table.py

**Review by kgunga — CHANGES_REQUESTED** [review]

File: infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py (hunk 1 above)
Comment (Git-Uzair) [reply infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py]: done

File: infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py (hunk 2 above)
Comment (Git-Uzair) [reply infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py]: Done

File: infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py (hunk 3 above)
Comment (Git-Uzair) [reply infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py]: Done

**Review by kgunga — APPROVED** [review]
👍

**Pull request merged**